manticore/manticore.py

```diff
@@ -22,37 +22,40 @@
 from .core.parser import parse
 from .core.smtlib import solver, Expression, Operators, SolverException, Array, ConstraintSet
 from core.smtlib import BitVec, Bool
-from .platforms import linux, decree, windows
+from .platforms import linux, decree, windows, binja
 from .utils.helpers import issymbolic
 from .utils.nointerrupt import WithKeyboardInterruptAs
 logger = logging.getLogger('MANTICORE')
 
-
-
+def makeBinja(program):
```

Comment:
Added makeBinja as well as a disassembler argument for the different platforms, so that
different disassemblers can be used